language/move-prover/tests/sources/stdlib/modules/libra_account.move

```diff
@@ -569,6 +581,59 @@ module LibraAccount {
         let transaction_fee_balance = borrow_global_mut<Balance<LBR::T>>(0xFEE);
         Libra::deposit(&mut transaction_fee_balance.coin, transaction_fee);
     }
+//    spec fun epilogue { // TODO: Verify this spec. Currently, unable to verify due to the multiplication involved
```

Comment:
@shazqadeer, here is an concrete example where multiplication is an issue. If there was no multiplication, it should work (please see the other function `simplified_epilogue`).